README.md

```diff
@@ -92,16 +219,623 @@ impl Deserr<MyError> for Name {
+#[derive(Deserr, Debug, PartialEq, Eq)]
+#[deserr(where_predicate = A: Deserr<__Deserr_E>)]
+struct Search<A> {
+    query: A,
+    limit: usize,
 }
 ```
 
+### Comparison with serde
+
+Since deserr needs to first deserialize the payload into a generic `Value` that allocates
+a lot of memory before creating your structure, it's a lot slower than serde.
+
+For example, at meilisearch for our search route, in case of a valid payload, we observed a 400% slowdown (4 times slower).
+That made our search request deserialize in 2us instead of 500ns.
+This is fast enough for most use cases but could be an issue if most of your time is spent deserializing.
+
+#### Datastructure support
+
+| datastructure       | serde | deserr | note |
+|---------------------|-------|--------|------|
+| Struct              |  yes  |  yes   |      |
+| Tuple struct        |  yes  |  no    |      |
+| Untagged Enum       |  yes  |  no    |      |
+| Untagged unit Enum  |  yes  |  yes   |      |
+| Tagged Enum         |  yes  |  yes   |      |
+
+#### Container attributes
+
+| features            | serde | deserr | note |
+|---------------------|-------|--------|------|
+| rename              |  yes  |  no    |      |
+| rename_all          |  yes  |  yes   |      |
+| deny_unknown_fields |  yes  |  yes   | With deserr you can call a custom function when an unknown field is encountered |
+| tag                 |  yes  |  yes   |      |
+| tag+content         |  yes  |  no    |      |
+| untagged            |  yes  |  no    | it's only supported for unit enums |
+| bound               |  yes  |  no    | Can be emulated with `where_predicate` |
+| default             |  yes  |  no    |      |
+| remote              |  yes  |  no    |      |
+| transparent         |  yes  |  no    |      |
+| from                |  yes  |  no    |      |
+| try_from            |  yes  |  yes   |      |
+| into                |  yes  |  no    |      |
+| crate               |  yes  |  no    |      |
+| validate            |  no   |  yes   | Allows you to validate the content of struct **after** it has been deserialized |
+| error               |  no   |  yes   | Specify the error type that should be used while deserializing this structure |
+| where_predicate     |  no   |  yes   | Let you add where clauses to the generated `Deserr` implementation |
+
+#### Field attributes
+
+| features            | serde | deserr | note |
+|---------------------|-------|--------|------|
+| rename              |  yes  |  no    |      |
+| alias               |  yes  |  no    |      |
+| default             |  yes  |  yes   |      |
+| flatten             |  yes  |  no    | serde doesn't support flattening + denying unknown field |
+| skip                |  yes  |  yes   |      |
+| deserialize_with    |  yes  |  no    | But it's kinda emulated with `from` |
+| with                |  yes  |  no    |      |
+| borrow              |  yes  |  no    |      |
+| bound               |  yes  |  no    |      |
```

Comment:
For `borrow`: can we say that `deserr` does not support types with references? I haven't actually checked, but I can't see how it would work.